kotlinx-coroutines-core/src/main/kotlin/kotlinx/coroutines/experimental/sync/Mutex.kt

```diff
@@ -100,6 +100,8 @@ public interface Mutex {
 
 /**
  * Creates new [Mutex] instance.
+ * The mutex created is fair: lock is granted in sequential order to the applicants.
```

Comment:
The technical name of this guarantee is "first come, first served" (FCFS), so the better way of saying it is that "lock is granted in first come, first served order"